201-vm-efficientip-vhd/azuredeploy.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "dnsNameForPublicIP": {
+            "value": "GEN-UNIQUE"
+        },
+        "existingVirtualNetworkName": {
+            "value": "GET-PREREQ-existingVnetName"
+        },
+        "existingVirtualNetworkResourceGroup": {
+            "value": "GET-PREREQ-existingVnetRG"
+        },
+        "subnetName": {
+            "value": "GET-PREREQ-subnetName"
+        },
+		"existingSecurityGroupName": {
+            "value": "GET-PREREQ-existingSecurityGroupName"
+        },
+        "osDiskVhdUri": {
+            "value": "GEN-SPECIALIZED-WINVHD-URI"
+        },
+        "vmName": {
+            "value": "GEN-UNIQUE-8"
+        },
+        "osType": {
```

Comment:
remove these last 3 and just use defaultValues

Reply:
Understand this one.  Changes has been made